packages/compass-query-history/src/stores/favorite-list-store.js

```diff
@@ -50,6 +54,7 @@ const configureStore = (options = {}) => {
     },
 
     runQuery(query) {
+      track('Query History Favorite Used');
```

Comment:
I was seeing this event when I clicked on a recent query in the query history. I think it might be a bug in that it's running both as a recent query and a favorite (these changes just show us that xD). Did you see similar? I can file a ticket for if so.

Reply:
Thank you for noticing! Opened https://jira.mongodb.org/browse/COMPASS-5243 and left some very rough code here to (mostly) only track one of these events.